### ISSUE TITLE
Cannot open any HDR video file

### ISSUE THREAD

**Issue by iTim314** [arch1t3cht/Aegisub#85]
I'm running Release 09 on macOS. I had this same issue with the abandoned/"official" Aegisub build for macOS.

When opening any HDR video encoded with Handbrake (default presets), I get the following error:

```
**Error loading file**
Could not open /path/to/file.mp4: 
YUV4MPEG: video is not in a supported format. 
FFmpegSource: Unknown video color space
```

The video plays fine in video players (QuickTime, VLC, etc.). The color profile is 9-16-9. If I open a non-HDR video with a color profile of 1-1-1, it opens fine.

**Comment by arch1t3cht** [comment]
Can you try a more recent versions from the actions page? Those have a newer version of ffms linked.

**Comment by iTim314** [comment]
@arch1t3cht I could not find a macOS build under Release 10. I tried building it myself but kept getting compile errors, so I just went with Release 09.

**Comment by arch1t3cht** [comment]
Yes, but there are prebuilt versions on the actions page https://github.com/arch1t3cht/Aegisub/actions

**Comment by iTim314** [comment]
Thanks for that. However, the issue is still there, although the error message has changed:

```
**Exception in event handler**
An unexpected error has occurred. Please save your work and restart Aegisub.

Error Message: vector
```

**Comment by iTim314** [comment]
This issue persists in Release 11.

**Comment by arch1t3cht** [comment]
This should be fixed now as a byproduct of 8563a6c06e67a54f7890823d9a137cb5e645f309 . HDR files can still look very strange in Aegisub since it does absolutely no gamut mapping or tone mapping (and just interprets the colors as sRGB or something similar), but at least it will not refuse entirely to load HDR video, so that you can at least work on subtitles on the video.
